doc/Output JSON.json

```diff
@@ -322,22 +351,22 @@
           "text": "Menstrual cycle"
         },
         "entry": [ {
-          "reference": "Observation/8973a466-6dc4-4c6c-8e04-aac141f3bddd"
+          "reference": "Observation/8da7eab5-c5c6-49d6-810f-aee45f2d80aa"
         } ]
       }, {
         "title": "Investigation Advice",
         "entry": [ {
-          "reference": "#1"
+          "reference": "ServiceRequest/f0d447b4-21b8-4753-a2d7-b3b8f1bf9ac2"
         } ]
       }, {
         "title": "Investigation Advice",
         "entry": [ {
-          "reference": "#2"
+          "reference": "ServiceRequest/f38f5202-cc57-4b97-affe-3769aa98c86d"
         } ]
       }, {
         "title": "Surgical Summary With Post OP Course",
```

Comment:
Surgical summary should not be in OP Consult record. But if recorded for past history - please add to section of "past medical history"